CMakeLists.txt

```diff
@@ -206,20 +211,15 @@ if(CMAKE_CURRENT_SOURCE_DIR STREQUAL "${CMAKE_CURRENT_BINARY_DIR}")
                         "initialize the build configuration.")
 endif()
 
-# Set module path appropriately.
-set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-
-# Set binary output path if not defined by user.
-if(EXECUTABLE_OUTPUT_PATH STREQUAL "")
-  set(EXECUTABLE_OUTPUT_PATH "${CMAKE_CURRENT_BINARY_DIR}/bin")
-endif()
-
-# Set library output path if not defined by user, but always set library output
-# path to binary output path for Xcode projects.
-if(CMAKE_GENERATOR STREQUAL "Xcode")
-  set(LIBRARY_OUTPUT_PATH "${EXECUTABLE_OUTPUT_PATH}")
-elseif(LIBRARY_OUTPUT_PATH STREQUAL "")
-  set(LIBRARY_OUTPUT_PATH "${CMAKE_CURRENT_BINARY_DIR}/lib")
+if(NOT XCODE)
+  # set binary output path if not defined by user
+  if(NOT EXECUTABLE_OUTPUT_PATH)
+    set(EXECUTABLE_OUTPUT_PATH "${CMAKE_CURRENT_BINARY_DIR}/bin")
+  endif()
+  # set library output path if not defined by user
+  if(NOT LIBRARY_OUTPUT_PATH)
+    set(LIBRARY_OUTPUT_PATH "${CMAKE_CURRENT_BINARY_DIR}/lib")
+  endif()
```

Comment:
Please remove these lines. This fails the coverage test now, because `caf-net-test` lands in `${CMAKE_CURRENT_BINARY_DIR}/bin`. This got copied over from the main repo, but here we didn't set `EXECUTABLE_OUTPUT_PATH` and `LIBRARY_OUTPUT_PATH` before. I think we shouldn't start doing it either. If anything, we should drop these lines from the main repo as well.